bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/scheduling/Strand.java

```diff
@@ -71,15 +70,14 @@
     public boolean blockedOnExtern;
     public Set<ChannelDetails> channelDetails;
     public Set<SchedulerItem> dependants;
-    public ObserverContext observerContext;
     public boolean cancel;
 
     SchedulerItem schedulerItem;
     List<WaitContext> waitingContexts;
     WaitContext waitContext;
     ItemGroup strandGroup;
 
-    private Map<String, Object> globalProps;
+    public Map<String, Object> globalProps;
```

Comment:
Shall we try not to make public variables? We can use getters for future variables. I know this class already uses a lot of public variables, but we can try to reduce them in the future

Reply:
We will hide this from others using java 9 modules. But better to keep things public here, as long as they are only accessed by us (runtime module) for performance reasons. what do you think?

Reply:
will fix.